.gitignore

```diff
@@ -1,4 +1,5 @@
 # MacOS files
 .DS_Store
 # Developer-specific IDE settings
-.vscode/settings.json
\ No newline at end of file
+.vscode/settings.json
+app/tests/var/*
```

Comment:
The test suite is capable of writing the rotated document images to disk (the `app/tests/var` directory) for visual comparison. [This logic is currently commented out](https://github.com/DSACMS/iv-doc-uploader/pull/11/files#diff-36b38b83fc11ae6dffca708ec2150e49bc3701baa03b708c85e4380db554e0dbR47), but may be useful when testing/developing locally.